README.md

```diff
@@ -698,7 +724,7 @@ You earn rewards for: Sharders and Miners
 
```

Comment:
Please add output sample to the collect-reward command

Reply:
this has been added